tests/Unit/Evolution/Systems/NewtonianEuler/Test_TurbulenceComputeItems.cpp

```diff
@@ -0,0 +1,177 @@
+  TestHelpers::db::test_compute_tag<
+      NewtonianEuler::Tags::RamPressureCompute<DataType, Dim>>("RamPressure");
+  TestHelpers::db::test_compute_tag<
+      NewtonianEuler::Tags::SpecificKineticEnergyCompute<DataType, Dim>>(
+      "SpecificKineticEnergy");
+
+  const auto box = db::create<
+      db::AddSimpleTags<NewtonianEuler::Tags::MassDensity<DataType>,
+                        NewtonianEuler::Tags::Velocity<DataType, Dim>,
+                        NewtonianEuler::Tags::SpecificInternalEnergy<DataType>,
+                        hydro::Tags::EquationOfState<EquationOfStateType>>,
+      db::AddComputeTags<
+          NewtonianEuler::Tags::SoundSpeedSquaredCompute<DataType>,
+          NewtonianEuler::Tags::SoundSpeedCompute<DataType>,
+          NewtonianEuler::Tags::InternalEnergyDensityCompute<DataType>,
+          NewtonianEuler::Tags::KineticEnergyDensityCompute<DataType, Dim>,
+          NewtonianEuler::Tags::MachNumberCompute<DataType, Dim>,
+          NewtonianEuler::Tags::RamPressureCompute<DataType, Dim>,
+          NewtonianEuler::Tags::SpecificKineticEnergyCompute<DataType, Dim>>>(
+      mass_density, velocity, specific_internal_energy, equation_of_state);
+
+  const auto expected_sound_speed_squared = NewtonianEuler::sound_speed_squared(
+      mass_density, specific_internal_energy, equation_of_state);
+  const auto expected_sound_speed =
+      Scalar<DataType>{sqrt(get(expected_sound_speed_squared))};
+  const auto expected_internal_energy_density =
+      NewtonianEuler::internal_energy_density(mass_density,
+                                              specific_internal_energy);
+  const auto expected_kinetic_energy_density =
+      NewtonianEuler::kinetic_energy_density(mass_density, velocity);
+  const auto expected_mach_number =
+      NewtonianEuler::mach_number(velocity, expected_sound_speed);
+  const auto expected_ram_pressure =
+      NewtonianEuler::ram_pressure(mass_density, velocity);
+  const auto expected_specific_kinetic_energy =
+      NewtonianEuler::specific_kinetic_energy(velocity);
+  CHECK(db::get<NewtonianEuler::Tags::SoundSpeedSquared<DataType>>(box) ==
+        expected_sound_speed_squared);
+  CHECK(db::get<NewtonianEuler::Tags::SoundSpeed<DataType>>(box) ==
+        expected_sound_speed);
+  CHECK(db::get<NewtonianEuler::Tags::InternalEnergyDensity<DataType>>(box) ==
+        expected_internal_energy_density);
+  CHECK(db::get<NewtonianEuler::Tags::KineticEnergyDensity<DataType>>(box) ==
+        expected_kinetic_energy_density);
+  CHECK(db::get<NewtonianEuler::Tags::MachNumber<DataType>>(box) ==
+        expected_mach_number);
+  CHECK(db::get<NewtonianEuler::Tags::RamPressure<DataType, Dim>>(box) ==
+        expected_ram_pressure);
+  CHECK(db::get<NewtonianEuler::Tags::SpecificKineticEnergy<DataType>>(box) ==
+        expected_specific_kinetic_energy);
+}
+
+template <size_t Dim, typename DataType>
+void test_compute_items(const DataType& used_for_size) noexcept {
```

Comment:
I'm quite confused by this test:
- it is testing SoundSpeedSquared, but that is already tested in Test_SoundSpeedSquared... the code looks copy-pasted here? Is this intentional?
- do the lines `// CHECK implementation of MachNumber here` mean there is a test you were still planning to add?
- you test that each free function (e.g., mach_number) produces the same output as the free function called through the databox (`MachNumberCompute`), but many of the free functions are not tested to have the correct result, so testing self-consistency doesn't seem useful. Could these missing correctness tests be added?

Reply:
- The lines 

```
TestHelpers::db::test_compute_tag<
      NewtonianEuler::Tags::SoundSpeedSquaredCompute<DataType>>(
      "SoundSpeedSquared");
TestHelpers::db::test_compute_tag<
      NewtonianEuler::Tags::SoundSpeedCompute<DataType>>("SoundSpeed");
```

are the same as in the file you mention. At some point my intention was to fuse both files into a single one. Would that make more sense, or should I just delete the lines above if you find it overkill to leave them here too?

- You are right--in the end I opted not to test the Mach number as it's just a division. I can either add the proper test, or remove the comments.

- Line 115 tests the correct implementation of the specific kinetic energy free function. Same for line 117 and kinetic energy density, line 130 and ram pressure (expected implementation starts in line 122), line 145 and internal energy density with 1d EoS, and line 163 with 2d EoS. Additionally (and as a repeat, see the first bullet point), lines 139 and 155 check the sound speed squared implementation. Is this what seems to be missed? Also, now I'm leaning towards fusing SoundSpeedSquared.*pp to the *pp files in this PR again. Maybe also rename these files to simple ComputeItems.*pp or similar as is done in GH?

Thanks for taking a look.

Reply:
- I like the idea of merging the files (src and tests) for the SoundSpeedSquared and TurbulenceComputeItems into a single ComputeItems file, as you describe. Totally up to you if/when you'd like to merge these. But until the files are merged, I would favor removing the SoundSpeedSquared tests that are duplicated here. 
- Please test the Mach number, as simple as it may be :)
- Ah, I'd missed the tests of the energy density functions, thanks for setting me straight. I think once you add the mach_number test then everything will be tested, and the sound_speed_squared test can stay or go depending on your decision about merging.